client/tso_dispatcher.go

```diff
@@ -328,10 +328,10 @@ tsoBatchLoop:
 		case td.tsDeadlineCh <- dl:
 		}
 		// processRequests guarantees that the collected requests could be finished properly.
-		err = td.processRequests(stream, dc, batchController, done)
+		tbcConsumed, err := td.processRequests(stream, dc, batchController, done, td.useAsyncStream())
```

Comment:
This means that if we don't turn on the newly introduced feature, it will always work in a synchronized way, do I understand correctly?

Reply:
Yes. But let's hold this PR for now as the performance regression is currently very likely to be a misreport due to the unstable test environment.